sciLifeLab_utils/run_assemblies.py

```diff
@@ -89,54 +89,20 @@ def main(args):
             sample_files.remove(pair2_file)
             library += 1
         sample_YAML.close
-        if not hasattr(args, "email"):
-            args.email = None
-        submit_job(sample_YAML_name, args.global_config, sample_dir_name ,
-                pipeline, args.env, args.email, args.time, args.project,
-                args.threads, args.qos) # now I can submit the job to slurm
-        os.chdir(projectFolder)
 
+        #Run the job
+        all_modules = {
```

Comment:
well, the job submission code should be able to understand what module needs to load.. if you run the align one it needs to load (bwa, samtools). I got suspicious in seeing here a couple of hardcoded modules... why only these?